insights/tests/test_commandparser.py

```diff
@@ -0,0 +1,24 @@
+from insights.core import CommandParser
+from insights.tests import context_wrap
+from insights.parsers import ParseException
+import pytest
+
+CMF = "blah: Command not found"
+NO_SUCH_FILE = "/usr/bin/blah: No such file or directory"
+
+
+class MockParser(CommandParser):
+    def parse_content(self, content):
+        pass
+
+
+def test_command_not_found():
+    with pytest.raises(ParseException) as e:
+        MockParser(context_wrap(CMF))
+    assert "Command not found" in str(e.value)
+
+
+def test_no_such_file_or_directory():
+    with pytest.raises(ParseException) as e:
+        MockParser(context_wrap(NO_SUCH_FILE))
+    assert "No such file or directory" in str(e.value)
```

Comment:
It would probably be a good idea to test with multi-line files containing the bad words as well to make sure it allows them to parse OK.